test/helpers/splunk/splunk.go

```diff
@@ -0,0 +1,33 @@
+package splunk
+
+import (
+	"time"
+
+	"github.com/openshift/cluster-logging-operator/test/framework/functional"
+	. "github.com/onsi/gomega"
+)
+
+// WaitOnSplunk waits for Splunk to be ready by checking HEC health and service status
+func WaitOnSplunk(f *functional.CollectorFunctionalFramework) {
+	time.Sleep(20 * time.Second)
```

Comment:
Do we still require the sleeps here and below?  Isn't the intent here to poll and not require a sleep?

Reply:
Yes we do still need the sleeps for splunk to get setup. The polling of read/write returns an error if the splunk server isn't up which fails the test.